### ISSUE TITLE
NTR: Flow cytometry: NK cells increased (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8745]
New term request Flow cytometry: NK cells increased (elthat_110211030610): Natural killer cells

**Comment by pnrobinson** [comment]
@MickeySegal
I see that HPO does not have a term for increased NK cell count
[Abnormal natural killer cell count HP:0040089](https://hpo.jax.org/app/browse/term/HP:0040089)
Is this request essentially for
**Elevated natural killer cell count**
in the blood circulation?

**Comment by MickeySegal** [comment]
Correct.  
https://www.omim.org/entry/618495
"Laboratory studies show increased NK cells that show impaired differentiation, as well as abnormal T cell populations or responses."

For the reverse we have used https://hpo.jax.org/app/browse/term/HP:0040219

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000677
- label: Elevated natural killer cell count
- parents: [Abnormal natural killer cell count (HP:0040089)](https://hpo.jax.org/app/browse/term/HP:0040089)
- pmid: 
- def: The count of natural killer cells per volume of blood is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8745
- synonyms:
